geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/RasterRDD.scala

```diff
@@ -89,13 +94,18 @@ abstract class TileRDD[K: ClassTag] {
 
   def reclassifyDouble(
     doubleMap: java.util.Map[Double, Double],
-    boundaryType: String
+    boundaryType: String,
+    ignoreNoData: Boolean = false
```

Comment:
Since the default value is already set for `ignoreNoData` in python, is there a point to having a default here?